packages/expo-cli/src/exp.ts

```diff
@@ -30,6 +30,7 @@ import {
   ProjectUtils,
   UserManager,
 } from 'xdl';
+import UnifiedAnalytics from 'xdl/build/UnifiedAnalytics';
```

Comment:
I think you should be able to just import it from `xdl`. Using `xdl/build/UnifiedAnalytics` might be risky, since it leaks package-specific structure into the `expo-cli`.